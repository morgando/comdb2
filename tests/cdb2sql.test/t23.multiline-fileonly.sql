exec procedure sys.cmd.send('dummy command');
exec procedure sys.cmd.send('dummy command');
