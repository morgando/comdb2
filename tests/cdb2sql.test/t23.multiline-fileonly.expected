(out='Non-registered tunable 'dummy'.')
(out='[ERROR] Unknown command <dummy>')
(out='Non-registered tunable 'dummy'.')
(out='[ERROR] Unknown command <dummy>')
